lib/mpl_toolkits/axes_grid1/parasite_axes.py

```diff
@@ -175,6 +179,10 @@ def apply_aspect(self, position=None):
 @functools.lru_cache(None)
 def parasite_axes_auxtrans_class_factory(axes_class=None):
     if axes_class is None:
+        cbook.warn_deprecated(
+            "3.3", message="Support for passing None to "
+            "parasite_axes_auxtrans_class_factory is deprecated; explicitly "
+            "pass the default Axes class instead.")
```

Comment:
Why not be explicit:
`...; explicitly pass ParasiteAxes instead`.

Same for the other warnings.

Reply:
Done.  For the others the default is Axes (well, whatever Axes is in that current module).